python/xorbits/_mars/dataframe/datasource/read_parquet.py

```diff
@@ -504,7 +564,9 @@ def _tile(cls, op: "DataFrameReadParquet"):
     def _execute_partitioned(cls, ctx, op: "DataFrameReadParquet"):
         out = op.outputs[0]
         engine = get_engine(op.engine)
-        with open_file(op.path, storage_options=op.storage_options) as f:
+        with fsspec.get_fs_token_paths(op.path, storage_options=op.storage_options)[
+            0
+        ].open(op.path, storage_options=op.storage_options) as f:
```

Comment:
Is it possible to use the high level API `fsspec.open` instead of `fsspec.get_fs_token_paths(...)[0].open`?

Reply:
The value returned by `fsspec.open` is a value with type `OpenFile`, which will cause the error `TypeError: expected str, bytes or os.PathLike object, not OpenFile` at `file = fastparquet.ParquetFile(f)` of function `read_to_pandas`

Reply:
Have you tried the

```python
of = fsspec.open(url, ...)
# of is just a place-holder
with of as f:
    # f is now a real file-like object holding resources
    f.read(...)
```

https://filesystem-spec.readthedocs.io/en/latest/features.html#openfile-instances

Reply:
I have tried it and fount it work, so I have used it partially